Makefile

```diff
@@ -63,6 +65,16 @@ build:
 regression:
 	$(MAKE) -C examples $@ 
 
+docker-regression:
+	docker run \
+	       -v $(realpath $(shell pwd))/../:/bsg_bladerunner \
+	       --env BSG_PLATFORM=dpi-verilator \
+	       --env BSG_DOCKER=1 \
+	       --env RISCV_BIN_DIR=/usr/bin \
```

Comment:
Same for RISC-V Tools. I would rather use BSG_MANYCORE as the path, than use /usr/bin

Reply:
Same as with Verilator, BSG_MANYCORE lives in the user's filesystem, so this would be a bit difficult. We could again keep the RISCV_BIN_DIR in something like `/riscv_install` or similar.